SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditor.cpp

```diff
@@ -29,6 +30,63 @@ DEFINE_LOG_CATEGORY(LogSpatialGDKEditor);
 
 #define LOCTEXT_NAMESPACE "FSpatialGDKEditor"
 
+namespace 
+{
+
+bool CheckAutomationToolsUpToDate()
```

Comment:
Feels strange this is in the `SpatialGDKEditor.cpp` any particular reason?

Reply:
That's the only place where where I need it so far.